Utilities/Remove-OldAzureVMs.ps1

```diff
@@ -164,16 +165,47 @@ if ($RemoveAutoCleanupVMs) {
 $cleanupRGScriptBlock = {
     $RGName = $args[0]
     $DryRun = $args[1]
+    $RemoveDisks = $args[2]
     if (-not $DryRun) {
+        if ($RemoveDisks) {
+            $vms = Get-AzVM -ResourceGroupName $RGName
+        }
         Remove-AzResourceGroup -Name $RGName -Verbose -Force
+
+        if ($? -and $RemoveDisks) {
+            foreach ($vm in $vms) {
+                $osDiskUri = $vm.StorageProfile.OSDisk.Vhd.Uri
+                if ($osDiskUri) {
```

Comment:
Please give an example of the Url. Otherwise we are not able to understand the magic indexes used below.